nevergrad/optimization/lamcts/MCTS.py

```diff
@@ -30,10 +30,16 @@ def __init__(self, lb, ub, dims, ninits, func):
         self.samples                 =  []
         self.nodes                   =  []
         self.Cp                      =  5
-        self.lb                      =  lb
-        self.ub                      =  ub
+        assert (lb is None) == (ub is None), "We will deal with one-sided bounds later."
+        self.lb                      =  lb if lb is not None else -(np.pi/2)*np.ones(dims)
+        self.ub                      =  ub if ub is not None else (np.pi/2)*np.ones(dim)
         self.ninits                  =  ninits
-        self.func                    =  func
+        def nevergrad_func(x):
+            y = np.tan(x)
+            return func(y)
```

Comment:
what is this for?
Also, avoid inline functions, that would prevent pickling.

Reply:
LaMCTS is not prepared for unbounded cases.